linmod/eval.py

```diff
@@ -31,3 +29,11 @@ def proportions_mae(samples, data):
         .group_by("lineage", "division", "day")
         .agg(mae=pl_mae("phi", "phi_sampled"))
```

Comment:
```suggestion
        .agg(mae=pl_mae("phi", "phi_sampled"))
        .group_by("division", "day")
        .agg(pl.sum("mae"))
```

Reply:
I want to rename this to "per division-day" like we mentioned, but to do that, I need to aggregate over lineages. I actually do this a little more cleanly in a future PR, but this should be good for here?

Reply:
(Batch this with below)